RuntimeUnityEditor/Utils/UI/InterfaceMaker.cs

```diff
@@ -56,33 +60,88 @@ private static GUISkin CreateSkin()
             // Reflection because unity 4.x refuses to instantiate if built with newer versions of UnityEngine
             var newSkin = typeof(Object).GetMethod("Instantiate", BindingFlags.Static | BindingFlags.Public, null, new []{typeof(Object)}, null).Invoke(null, new object[] {GUI.skin}) as GUISkin;
             Object.DontDestroyOnLoad(newSkin);
-
+            _fontSize = RuntimeUnityEditorCore._initSettings.RegisterSetting("General", "font Size", 14, "Controls font size on tool's UI.");
+            _fontName = RuntimeUnityEditorCore._initSettings.RegisterSetting("General", "font Name", "Arial.ttf", "Controls font used on tool's UI, should be dynamic if used with size.");
```

Comment:
Any reason why these are in General and not in Skin? Also the names start lowercase unlike other settings.

Reply:
The Skin Section is supposed to be the can of worms only people interested in skinning the tool to a specific game would go there and make changes, while font options are an accessibility thing most fitting to General, I placed them at Global at first but then fixed it to go with your naming conventions.

Reply:
Lowercase names are my bad, got confused with the convention, I can fix this if you want to, or eventually just send the font options part, if making a Skin section is not on your roadmap.